human-forces-provider/CMakeLists.txt

```diff
@@ -56,3 +56,5 @@ add_warnings_configuration_to_target(TARGETS ${PROJECT_NAME} PRIVATE)
 
 target_include_directories(${PROJECT_NAME} SYSTEM PRIVATE ${EIGEN3_INCLUDE_DIR})
 
+yarp_install(FILES app/conf/HumanForcesProvider.ini DESTINATION ${YARP_CONTEXTS_INSTALL_DIR}/human-dynamic-estimation)
```

Comment:
Do you mind changing the name of the file to be `human-forces-provider.ini` to be more consistent with the rest of the repository?